public/wp-content/themes/base-theme/app/Theme.php

```diff
@@ -238,39 +167,5 @@ public function load_options_panel()
             'menu_title'    => 'Javascript / CSS',
             'parent_slug'   => 'theme-options-settings',
         ));
-
-        
-
-    }
-
-    public function set_menus()
```

Comment:
set_menus was renamed to `loadMenus`
set_image_size was refacored and now is substituted with `loadThumbnailSupport`